cmd/openshift-install/agent.go

```diff
@@ -70,6 +71,21 @@ var (
 		},
 	}
 
+	agentConfigImageTarget = target{
+		name: "Agent Config Image",
+		command: &cobra.Command{
```

Comment:
@zaneb Should this command also be hidden?

Reply:
Per discussion with Zane, this command will be available for the customer to run and should not be hidden.